src/ports/postgres/modules/recursive_partitioning/decision_tree.py_in

```diff
@@ -1582,38 +1578,17 @@ def _create_summary_table(
 # ------------------------------------------------------------
 
 
-def _get_filter_str(schema_madlib, cat_features, con_features,
-                    boolean_cats, dependent_variable,
-                    grouping_cols, max_n_surr=0):
+def _get_filter_str(dependent_variable, grouping_cols):
```

Comment:
I think this function is still called with 7 arguments in the random forest module.